docs/content/users/basics/architecture.md

```diff
@@ -0,0 +1,109 @@
+# How DDEV Works
+
+DDEV is a [Go](https://go.dev) application that stores its configuration in [files on your workstation](#directory-tour).  
```

Comment:
Think we need to mention golang here? Not generally sure people normally need to know that.

Reply:
I do! It’s an essential part of what DDEV is, which is perfect for this page. Just like the “Look But Don’t Touch” section, it’s about getting comfortable with the tool and what it does.